### DISCUSSION TITLE
WARNING: Ignoring https://dl-cdn.alpinelinux.org/alpine/v3.16/main: Permission denied

### DISCUSSION THREAD

**Discussion by vicjoe** [nextcloud/all-in-one#1511] [Questions] [answered]
### How to use GitHub
* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are interested into the same feature.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

When start the container nextcloud_aio_nextcloud, 
It is very slow for step "Installing imagemagick via apk..."
and finally failed for "Installing PHP build dependencies..."

So add a proxy for the domain "dl-cdn.alpinelinux.org"
then it will be very quick, however get the failure below

_Installing imagemagick via apk...
Installing PHP build dependencies...
WARNING: Ignoring https://dl-cdn.alpinelinux.org/alpine/v3.16/main: Permission denied
ERROR: unable to select packages:
Could not install build-deps!
Installing Imagick via PECL...
error: 'imagick' does not exist_

### Describe the solution you'd like

Search on the internet, seems there is solution below
_RUN sed 's/https/http/g' -i /etc/apk/repositories
RUN apk update
RUN apk add --no-cache bash_

Can we also add this in the startup.sh file before installing the php extensions ?
or some other better solution ? Thanks 
@szaimen

**Comment by szaimen** [comment, marked as answer]
This looks like a dns issue on your side based on e.g. https://github.com/docker-library/openjdk/issues/402#issuecomment-611604736 so you should rather fix this yourself. Also reducing the security by switching from https to http is not something we will allow.

**Reply by vicjoe** [reply]
Restart the nextcloud in the morning time, then the permission error is gong, 
Seems the servers of alpine CDN is not stable, and DNS could also resolve this problem.
Thanks a lot for the direction ~
